### ISSUE TITLE
Some question about continue training

### ISSUE THREAD

**Issue by wtmlon** [Gaffey/ExCP#7]
Dear ExCP authors,
 * We are currently attempting to follow your work. We try to use ExCP on Llama3-8B SFT training, and finding that after continuing the train from quantized checkpoint, the loss becomes abnormal.
 *  I just wanna know, do you have similar problem and how to fix that, and do you have PanGu-π-7B continue training loss figures?
![image](https://github.com/user-attachments/assets/7c9ccda7-9b31-4a02-b0fc-9a92505ada4f)

Thank you for your help!

Best regards.

**Comment by Peter00796** [comment]
I guess it is normal for loss to increase when training from a quantized checkpoint since pruning and quantizing will certainly result in precision loss and certain features missing of the parameters, especially for this lossy way of quantization (they might prune too much). 
And it takes iterations for the loss to drop.

**Comment by Gaffey** [comment]
The loss would increase after the resuming in our training, but not that much. Maybe you can try to modify the hyperparameter `prune_alpha` and `prune_beta` for compression.
